jmxtrans/docker/README.md

```diff
@@ -0,0 +1,74 @@
+# JmxTrans Docker image
+
+<img src="http://www.jmxtrans.org/assets/img/jmxtrans-logo.gif"/>
+
+Connecting the outside world to the JVM.
+
+This is a fully functional JmxTrans application instance, based on the last releaase.
+[http://www.jmxtrans.org/).
+
+
+# Usage
+
+```
+docker run -d -v `pwd`/json-files:/var/lib/jmxtrans jmxtrans/jmxtrans 
+docker run -d -v `pwd`/json-files:/var/lib/jmxtrans jmxtrans/jmxtrans start 
+docker run -d -v `pwd`/json-files:/var/lib/jmxtrans -P jmxtrans/jmxtrans jmx
+docker run -d -v `pwd`/json-files:/var/lib/jmxtrans -p 9999:2101 jmxtrans/jmxtrans jmx
+```
+
+You have two commands available :
+* start (default value)  
+* jmx to start the jvm with jmx debug mode
```

Comment:
I enabled it by default and keep an option to disable it.
Documentation is updated.

Best regards,